content/scalability/docs/control-plane.md

```diff
@@ -240,4 +244,50 @@ spec:
 
 - This FlowSchema captures all list event calls made by service accounts in the default namespace.
 - The matching precedence 8000 is lower than the value of 9000 used by the existing service-accounts FlowSchema so these list event calls will match list-events-default-service-accounts rather than service-accounts.
-- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently. 
+- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently.
+
+## Use Shared Informers for Custom Controllers
+
+When building controllers and automation that integrate with the Kubernetes API you will often need to get information from Kubernetes resources. If you poll for these resources regularly it can cause a significant load on the API server.
+
+Using an [informer](https://pkg.go.dev/k8s.io/client-go/informers) from the client-go library will give you benefits of watching for changes to the resources based on events instead of polling for changes. Informers further reduce the load by using shared cache for the events and changes so multiple controllers watching the same resources do not add additional load.
```

Comment:
For multiple controllers its more efficient to be using shared informers, if we want to include that as well.

Reply:
All informers are shared informers AFAIKT. The type for genericInformer has a sharedInformerIndex. Do you know if there's something special that has to be done for shared and not shared informers?